### ISSUE TITLE
Feature Request: Video passthrough

### ISSUE THREAD

**Issue by MYodaTheRe** [RandomEngy/VidCoder#618]
Hi,

would it be possible to just add a checkbox in the profile settings, which would enable passthrough for videos just like the one for audio tracks does? I just want to create an .mkv file without reencoding. Of course I could use the makemkvtoolnix tools, but it isn't near handy as VidCoder is (especially when it comes to batching / creating of extra tracks for just forced subtitles etc.)

Best and thank you,
Michael

**Comment by RandomEngy** [comment]
VidCoder uses the HandBrake API to do its encoding. Unfortunately the HandBrake team has said that remuxing/direct video copy is not something they intend to add. You could always [ask them to add it](https://handbrake.fr/community.php).

**Comment by MYodaTheRe** [comment]
Ah okay, thank you for the answer! I know that Handbrake uses ffmpeg in the background and in VidCoder there is a cli-field in which you can put instructions which will get forwarded to the encoder. Would it be possible this way, if I put in there the video copy command?

**Comment by spikemixture** [comment]
I asked the same question and received the same reply about a passthrough. 
Why the "team" would have no intention only they would know...
I recommend vidcoder on many forums and sites and use it regularly.

I would love the option of remuxing mkv to mp4 !

**Comment by RandomEngy** [comment]
No, it's not possible. Handbrake compiles the DLL and exposes methods to call. Those do not include any video copy options. You can't access ffmpeg directly, and the video copy would need to be integrated directly into the HandBrake pipeline anyway.

**Comment by chschmit** [comment]
Move on from VidCoder and Handbrake...since they don't intend to incorporate video passthrough.

Use MyFFMPEG, or this script in a batch file (just drag/drop your files on the batch file):

echo off
:again
C:\ffmpeg.exe -i "%~1" -codec copy "%~p1%~n1.mp4"
shift
if "%~1" == "" goto:eof
goto:again

**Comment by Patrick1945** [comment]
To do video passtrough I use either DVDFab (Not Free) or Avidemux (Free) and that works fine.
